packages/react/src/components/CommandBar/CommandBar.base.tsx

```diff
@@ -179,7 +179,7 @@ export class CommandBarBase extends React.Component<ICommandBarProps, {}> implem
       onClick: this._onButtonClick(item),
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28418&commitId=e139c4a27f52492093f55156ddd4d72e7207753f&env=prod&clientType=FLUENTUI)


  <details><summary><b>CommandBar</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|CommandBar.Text only.hover.chromium.png|69| Changed|
|CommandBar.Root.hover.chromium.png|69| Changed|
|CommandBar.Root - RTL.hover.chromium.png|69| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>